examples/README.md

```diff
@@ -0,0 +1,64 @@
+Creating a basic S2I builder image  
+---
+
+### Getting started  
+
+Directory skeleton:  
+
+* `Dockerfile` – standard Dockerfile where we’ll define the base builder image.  
+* `assemble` - responsible for building the application.  
+* `run` - responsible for running the application.  
+* `usage` - optional script responsible for printing the usage of the builder.  
+* `save-artifacts` - optional script for incremental builds that save built artifacts.  
+
+We are going to start by adding our logic to the [`Dockerfile`](nginx-app/Dockerfile).  
+
+In our [`assemble`](nginx-app/assemble) script we define other actions that need to be run when we want to install our application, this is where we build python modules or bundle install our required components, however in this example we leave it as it is. It will only override the default index.html file. It will also copy over any previously saved artifacts.  
+
+Let's create our [`run`](nginx-app/run) script that will start the application.  
+
+We can also specify a logic for an incremental build in [`save-artifacts`](nginx-app/save-artifacts).    
```

Comment:
Since you're now linking, this section needs some rewording, like:

"The first piece of the builder image is a Dockerfile that installs anything your image is going to need to build or run application code, you can see an example here[link]"

"Next we need an assemble script which....... the assemble script for this example is found here[link].  This assemble script just overrides the default index.html, but normally you would do ........"

"The final required piece is a run script which starts the application.  The example run script can be seen here[link]"

"You can also optionally define a save-artifacts script which allows a build to reuse content from the last version of the application image that was built.  The sample save-artifacts script can be seen here[link]"

(please don't copy the above suggestions verbatim, they probably need more polish than what i spent time on, but hopefully you get the idea)